tools/nnpackage_tool/model2nnpkg/README.md

```diff
@@ -9,16 +9,16 @@ It takes `modelfile` as input and generates `nnpackage`.
 ```
 usage:  model2nnpkg.py [options]
   Examples:
-      model2nnpkg.py -m add.tflite                           => create nnpkg "add" in current directory
-      model2nnpkg.py -o out -m add.tflite                    => create nnpkg "add" in out/
-      model2nnpkg.py -o out -p addpkg -m add.tflite          => create nnpkg "addpkg" in out/
-      model2nnpkg.py -c add.cfg -m add.tflite                => create nnpkg "add" with add.cfg
-      model2nnpkg.py -o out -p addpkg -m a1.tflite a2.tflite => create nnpkg "addpkg" with models a1.tflite and a2.tflite in out/
+      model2nnpkg.py -m add.tflite                                              => create nnpkg "add" in current directory
+      model2nnpkg.py -o out -m add.tflite                                       => create nnpkg "add" in out/
```

Comment:
Thank you for keeping aligned. However, new example requires several spaces and it may requires users to scroll to see whole line. It may be better to keep old examples w/o additional spaces.

Reply:
OK, I will do that and change only the last example to two lines.